setup.py

```diff
@@ -19,10 +19,10 @@
 
 dev_requirements = [
     "tox==3.5.3",
-    "black==20.8b1",
+    "black==22.3.0",
 ]
 
-doc_requirements = ["mkdocs==1.0.4", "mkdocs-material==4.6.3"]
+doc_requirements = ["mkdocs==1.3.0", "mkdocs-material==8.2.9"]
```

Comment:
Why do you change this requirements in this PR? I'm not saying we have to keep old requirements, but these changes have nothing related to the initial objective of the PR. Maybe you can create another one (or at least split your PR into multiple commits).